README.md

```diff
@@ -1,8 +1,17 @@
 # Marama Editor
-## Required
-SBT
-Scala
-## Building
+
+## Development
+
+This project is based on Scala and sbt. To start developing you can import the project by opening the build.sbt file inside the project root with an IDE like IntelliJ or Eclipse.
+
+## Documentation
+
+It is recommended before you start development to read the technical design document that is provided in the source. 
+
+[TODO: describe how to build technical design document] 
```

Comment:
TODO

Reply:
Also read the devlog?

Reply:
True but that is not really a concern for this repo. De documentation is inside this repo so it should be read before starting development.

Reply:
I'd recommend fully removing the TODO, let them figure it out for themselves.

Reply:
Ok, removed.